### ISSUE TITLE
Industrial electrolyzer does not accept Almandine (and stops other dusts processing)

### ISSUE THREAD

**Issue by christianvdstap** [GTNewHorizons/GT-New-Horizons-Modpack#11544]
### Your GTNH Discord Username

Lalaru

### Your Pack Version

2.2.0.0

### Your Server

ETA

### Your Expectation

Almandine dust can be processed in the industrial electrolyzer

### The Reality

Almandine dust can not be processed in the industrial electrolyzer
(and it can in a normal electrolyzer, it is EV so even if it did 2 in parallel a HV output hatch for oxygen should be able to contain all the produced oxygen to my understanding 12*2 < 64).

Removing the almandine will let it process the other dusts.

### Your Proposal

Fix the bug please.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by miozune** [comment]
It actually does up to 8 parallel with EV. Closing as duplicate of #11448 - as a workaround, you can use EV Output Hatch.
